website/templates/index.html

```diff
@@ -110,7 +110,11 @@ <h4>1. How do I know my OSCN Case Number?</h4>
 			</div>
 			<div class="col-sm-6">
 				<h4>2. What if I still can't find my case number?</h4>
-				<p>Check with your public defender or search your name online at <a href=“www.oscn.net/dockets/search.aspx”>www.oscn.net/dockets/search.aspx </a> </p> </div> <div class="col-sm-6">
+				<p>Check with your public defender or search your name online at <a href="http://{{oscnRedirect}}">www.oscn.net/dockets/search.aspx </a> 
```

Comment:
Here's how to make the URL an absolute URL:

```suggestion
				<p>Check with your public defender or search your name online at <a href="https://www.oscn.net/dockets/search.aspx">www.oscn.net/dockets/search.aspx</a> 
```